lib/dotcom_web/templates/layout/root.html.heex

```diff
@@ -66,6 +66,22 @@
   </head>
   <%= content_tag(:body, class: Dotcom.BodyTag.class_name(@conn)) do %>
     <div class="body-wrapper" id="body-wrapper">
+      <%= if !Application.get_env(:dotcom, :is_prod_env?) && Gettext.get_locale(Dotcom.Gettext) != "en" do %>
+        <% completion_percent = Mix.Tasks.Gettext.Translate.completion_percent() %>
+        <% completion_ratio = Mix.Tasks.Gettext.Translate.completion_ratio() %>
```

Comment:
nit: Feels like this UI shouldn't be the responsibility of the mix task.

Reply:
Yeah, I am also using it in the translation task. There just isn't a better place to put things that we don't plan on sticking around for long. That's why I put multiple modules in that task file.

Reply:
Also, if you want something to be processed at compile time, you have to define the function in a different module and then set it as a module attribute somewhere else. This we want to define at compile time because it won't change and we don't want to recheck the template files every time we load a page.